src/wp-includes/media.php

```diff
@@ -6233,16 +6233,23 @@ function wp_high_priority_element_flag( $value = null ) {
  * @return string[] An array of mime type mappings.
  */
 function wp_get_image_editor_output_format( $filename, $mime_type ) {
+	$heic_images = array(
```

Comment:
maybe name this variable `default_output_format` or similar, it could be expanded beyond heic.

Reply:
Good idea, thanks!